src/components/CalendarModal/CalendarModal.module.scss

```diff
@@ -6,22 +6,66 @@
   height: 100%;
   background: rgba(0, 0, 0, 0.6);
   text-align: center;
-  z-index: 1;
+  z-index: 10;
 }
 
 .modal-main {
   padding: 1rem;
   position: fixed;
   background: white;
-  width: 80%;
+  max-width: 500px;
+  width: 60%;
   height: auto;
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
+  border-radius: 10px;
+}
+
+.modal-main button {
```

Comment:
is this line of code needed?

Reply:
The modal-main section - it looks like the background white is overwritten and unused

Reply:
oh, I added this .modal-main button section to turn the cancel button background white, otherwise it shows up gray and doesn't look as nice. I was going off of the design here: https://www.figma.com/file/iUqlIJecPtOMRY6wR2ng53/%5BAdditional-Feature%5D-Events-Page?node-id=801-1530